episode-6.msrt

```diff
@@ -387,7 +391,8 @@
 78
 00:05:06,020 --> 00:05:10,110
 [ru] [саркастически] Эх, радуйся, Кэррот. Похоже, никто не увидит твое чУдное зелье..
-[en] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
+[en] [Version 1:] [sarcastically] Ah, be glad Carrot. It seems nobody will see your wondrous potion...
```

Comment:
This is the best I've seen of this sentence, but it's still pretty formal and stilted. A sentence like this would tip me off that it was literally translated from something else.